code/game/dna/genes/monkey.dm

```diff
@@ -60,15 +57,16 @@
 		D.affected_mob = O
 		M.viruses -= D
 
-
-	for(var/obj/T in (M.contents-implants))
-		qdel(T)
-
 	O.loc = M.loc
 
 	if(M.mind)
 		M.mind.transfer_to(O)	//transfer our mind to the cute little monkey
 
+	if(!connected)
```

Comment:
It can still drop everything in there, once the patient gets taken out everything he had on him would be taken out with him